draft-pda-protocol.md

```diff
@@ -950,31 +956,68 @@ indefinitely until a suitable output is found.
 
 [TODO: Define Hits-specific protocol messages.]
 
-# System design
+# Operational Considerations {#operational-capabilities}
+
+Prio has inherent constraints derived from the tradeoff between privacy
+guarantees and computational complexity. These tradeoffs influence how
+applications may choose to utilize services implementing the specification.
 
-[[OPEN ISSUE: This section seems like a catch-all for things not in other
-sections. Perhaps there is a natural home for aggregator discovery, share
-uploading, open issues, and system parameters?]]
+## Protocol participant capabilities {#entity-capabilities}
 
-## Aggregator discovery
+The design in this document has different assumptions and requirements for
+different protocol participants, including clients, aggregators, and
+collectors. This section describes these capabilities in more detail.
 
-[[OPEN ISSUE: writeme]]
+### Client capabilities
 
-## Share uploading
+Clients have limited capabilities and requirements. Their only inputs to the protocol
+are (1) the PDAParam structure configured out of band and (2) a corresponding value to be
+submitted in a report. Clients are not expected to store any state across any upload
+flows, nor are required to implement any sort of report retry mechanism should it fail.
+By design, the protocol in this document is robust against individual client upload
+failures since the protocol output is an aggregate over all inputs.
```

Comment:
Also worth mentioning here: The aggregators validate inputs before consuming them.

Reply:
That's just par for the course -- I don't think it changes any of the capacity or operational requirements?